components/style/themes/default.less

```diff
@@ -264,3 +264,9 @@
 // ---
 @rate-star-color: #f5a623;
 @rate-star-bg: #e9e9e9;
+
+// Card
+// ---
+@card-head-height: 48px;
```

Comment:
Hmm the current class uses 'head' as name, just to make it consistent, should I change to header?

https://github.com/ant-design/ant-design/blob/feature-2.9/components/card/style/index.less#L22